test/utils.js

```diff
@@ -125,25 +126,33 @@ module.exports.extractInfoFromResponse = function (res) {
                 refreshTokenDomain = i.split(";")[1].split("=").slice(1).join("=");
             }
             refreshTokenHttpOnly = i.split(";").findIndex((j) => j.includes("HttpOnly")) !== -1;
-        } else {
-            idRefreshTokenFromCookie = i.split(";")[0].split("=")[1];
-            if (i.split(";")[2].includes("Expires=")) {
-                idRefreshTokenExpiry = i.split(";")[2].split("=")[1];
-            } else if (i.split(";")[2].includes("expires=")) {
-                idRefreshTokenExpiry = i.split(";")[2].split("=")[1];
-            } else {
-                idRefreshTokenExpiry = i.split(";")[3].split("=")[1];
-            }
-            if (i.split(";")[1].includes("Domain=")) {
-                idRefreshTokenDomain = i.split(";")[1].split("=")[1];
-            }
-            idRefreshTokenHttpOnly = i.split(";").findIndex((j) => j.includes("HttpOnly")) !== -1;
         }
     });
+
+    let refreshTokenFromHeader = undefined;
+    if (res.headers["st-refresh-token"]) {
+        const [value, expiry] = res.headers["st-refresh-token"].split(";");
+        refreshTokenFromHeader = {
+            value,
+            expiry: Number.parseInt(expiry),
+        };
+    }
+    let accessTokenFromHeader = undefined;
+    if (res.headers["st-access-token"]) {
+        const [value, expiry] = res.headers["st-access-token"].split(";");
+        accessTokenFromHeader = {
+            value,
+            expiry: Number.parseInt(expiry),
+        };
+    }
```

Comment:
I think this needs to be updated.

Reply:
You're right, I'll update it. Although the parsing still works since this will just parse it as NaN